### DISCUSSION TITLE
UI is slightly off when embedding in Sanity Studio

### DISCUSSION THREAD

**Discussion by mmgj** [tldraw/tldraw-v1#558] [General]
Hello folks. 
I am currently writing a tldraw plugin for [Sanity Studio](https://sanity.io). I've got everything working pretty smoothly except this minor issue; The UI seems to be positioned slightly off! As you can see in the enclosed screenshot the "styles"-menu is a handful of pixels off-screen and the menu items in the open "Edit" dialog seem to be overflowing as well. 

If I resize the width of the component, it reflows perfectly only still slightly off so it seems the component is still responsive. 

Any idea what could be causing this? I opted to post here first as I'm not sure if this is a problem for anyone but me, but I'll be happy to post an issue if that is more appropriate.

<img width="761" alt="Screenshot 2022-08-19 at 21 46 21" src="https://user-images.githubusercontent.com/174970/185697563-b57f8c8a-eb3c-4933-ba83-a64a20c287e8.png">

**Comment by judicaelandria** [comment]
thanks for the feedback will look into it
